pep-0655.rst

```diff
@@ -55,12 +55,25 @@ a mix of both required and potentially-missing keys:
        title: str
        year: NotRequired[int]
 
+This PEP also makes it possible to define TypedDicts in the
+:pep:`alternative syntax <589#alternative-syntax>`
+with a mix of required and potentially-missing keys,
+which is not currently possible at all because the alternative syntax does
+not support inheritance:
+
+::
+
+   Actor = TypedDict('Actor', {
+       'name': str,
+       'in': NotRequired[List[str]],  # "in" is a keyword, so needs alternative syntax
```

Comment:
I think "functional syntax" is clearer than "alternative syntax"

```suggestion
:pep:`alternative functional syntax <589#alternative-syntax>`
with a mix of required and potentially-missing keys,
which is not currently possible at all because the functional syntax does
not support inheritance:

::

   Actor = TypedDict('Actor', {
       'name': str,
       'in': NotRequired[List[str]],  # "in" is a keyword, so the functional syntax is necessary
```

Reply:
> I think "functional syntax" is clearer than "alternative syntax"

Agreed. However the term of art, as put forth in the earlier PEP 589, is ["alternative syntax"](https://www.python.org/dev/peps/pep-0589/#alternative-syntax), so I'm trying to remain consistent with it.

Thus I would advocate leaving the original wording for the most part. However I wouldn't mind changing the first mention in this paragraph from "alternative syntax" to "alternative functional syntax", leaving the other usages as-is. What do you think @AlexWaygood ?

Reply:
Hah, I think I've been hanging out in the [CPython test suite](https://github.com/python/cpython/blob/375a56bd4015596c0cf44129c8842a1fe7199785/Lib/test/test_typing.py#L4534) too much recently. (I've also seen it described [elsewhere](https://github.com/python/mypy/issues/2492#issuecomment-309157987) as the "functional syntax").

> I wouldn't mind changing the first mention in this paragraph from "alternative syntax" to "alternative functional syntax", leaving the other usages as-is.

Consistency with the original PEP is good — let's go with this 👍